projects/batfish-client/src/org/batfish/client/Client.java

```diff
@@ -333,12 +332,12 @@ private boolean answer(String questionTemplateName, String paramsLine,
     * type.
     * @throws BatfishException If the type encoded in input {@code value}
     * does not match the type specified in {@code variable}.
-    * @throws InvalidStateException If the input {@code value} is null.
+    * @throws IllegalArgumentException If the input {@code value} is null.
```

Comment:
Sorry -- I meant `IllegalStateException`, not `InvalidStateException`; and if you use [`java.lang.IllegalStateException`](https://docs.oracle.com/javase/7/docs/api/java/lang/IllegalStateException.html) instead of that `sun` InvalidStateException you won't have the compiler error you saw before.